app/mailers/notification_mailer.rb

```diff
@@ -1,11 +1,10 @@
 class NotificationMailer < ActionMailer::Base
-  default to: Proc.new { User.pluck(:email) },
-    from: "noreply@firehose-defcon.com"
- 
-  def send_notification(user)
-   @user = user
-    mail(subject: "A new post from #{@user.email} has been added to the Defcon Student Group")
-    
+  default from: "noreply@firehose-defcon.com"
+
+  def send_notification(new_post, recipient)
```

Comment:
I'm now passing two arguments: the post and each recipient: this way I can include a preview and customise the mail with the recipient's name.